### PR TITLE
Denis/yoext 92

### PR DESCRIPTION
**Pull request by Nebyt**
Added the window manager.

### CODE REVIEW

**Review by Nebyt — COMMENTED** [review]
/check

File: packages/yoroi-extension/features/support/windowManager.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    this.driver = driver;
+    this.windowHandles = [];
+  }
+
+  async init() {
+    const mainWindowHandle = await this.privateGetCurrentWindowHandle();
+    this.windowHandles.push({ title: 'main', handle: mainWindowHandle });
+  }
+
+  privateGetHandleByTitle(title: string): Array<CustomWindowHandle> {
```
Comment (zuzunker) [inline packages/yoroi-extension/features/support/windowManager.js]: use `_foo` convention for "private" functions plz

**Review by zuzunker — CHANGES_REQUESTED** [review]

File: packages/yoroi-extension/features/support/windowManager.js (hunk 1 above)
Comment (Nebyt) [reply packages/yoroi-extension/features/support/windowManager.js]: Fixed

**Review by yushih — APPROVED** [review]

**Comment by yushih** [comment]
Nice! It made the step definitions much cleaner!

**Pull request merged**